code/modules/minimap/minimap_objects/objects/info_map.dm

```diff
@@ -0,0 +1,23 @@
+/obj/info_map
+	name = "Information Map"
+	icon = 'icons/obj/decals/posters.dmi'
+	icon_state = "pw_map"
+	var/atom/movable/minimap_ui_handler/minimap_ui
+	var/obj/minimap/info/infomap
+
+/obj/info_map/New()
```

Comment:
disposing

Reply:
what changes would you recommend from the default obj ex_act?

Reply:
What changes from the default obj `ex_act` would you recommend?

Reply:
now that it's a machine, it's using the default machinery `ex_act`